adsmp/validate.py

```diff
@@ -164,7 +169,7 @@ def fields_match(self, bibcode, s1, s2, field):
                     'Bibcode {}: cite_read_boost field is different between databases. Old: {} New: {}'.format(bibcode, f1,
                                                                                                                f2))
                 return False
-            elif (abs(f1-f2)/f1) > 0.1:
+            elif (old_div(abs(f1-f2),f1)) > 0.1:
```

Comment:
The default python 3 behavior (or python 2 behavior when `from __future__ import division` is present) is what we want (i.e., 1/2 = 0.5) so we do not need old_div.